File: codingbat/sum13.java
public int sum13(int[] nums) {
  if (nums.length < 1) return 0;
  int sumNum = 0;
  for (int i = 0; i < nums.length; i++) {
    if (nums[i] == 13) {
      i++;
      continue;
    }
    sumNum += nums[i];
  }
  return sumNum;
}

File: codingbat/python/sum13.py
def sum13(nums):
  if len(nums) < 1: return 0
  total = 0
  for i in range(len(nums)):
    if nums[i] == 13 or (i > 0 and nums[i - 1] == 13): continue
    total += nums[i]
  return total

